Broken Makefile and lint test

Makefile uses python should be python3
and I got a lint error on Debian Bullseye and Ubuntu Focal Fossa:
python3 -m pylint check_http_json.py
************* Module check_http_json
check_http_json.py:614:0: E0012: Bad option value 'consider-using-with' (bad-option-value)

--- a/.github/workflows/unittest.yml
+++ b/.github/workflows/unittest.yml
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3,9]
+        python-version: [3.8, 3.11, 3.12]
     name: GitHub Action
     steps:
     - name: Checkout

--- a/Makefile
+++ b/Makefile
@@ -1,9 +1,11 @@
 .PHONY: lint test coverage
 
+PYTHON_PATH?=python3
+
 lint:
-	python -m pylint check_http_json.py
+	$(PYTHON_PATH) -m pylint check_http_json.py
 test:
-	python -m unittest discover
+	$(PYTHON_PATH) -m unittest discover
 coverage:
-	python -m coverage run -m unittest discover
-	python -m coverage report -m --include check_http_json.py
+	$(PYTHON_PATH) -m coverage run -m unittest discover
+	$(PYTHON_PATH) -m coverage report -m --include check_http_json.py

